### PR TITLE
Added two new questions for python quiz

### PR DESCRIPTION
**Pull request by Kunal-Garg-12**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,24 @@
 const pythonQuiz = [
+  {
+    Question: "In Python, which of the following method returns last value from a list?",
+    Answer: "pop()",
+    Distractor1: "remove()",
+    Distractor2: "last()",
+    Distractor3: "end()",
+    Explanation:
+      "By default, if there is no index specified, the pop() method will remove the last item that is contained in the list. Besides just removing the item, pop() also returns it.",
+    Link: "https://www.freecodecamp.org/news/python-pop-how-to-pop-from-a-list-or-an-array-in-python/#howtousethepopmethodwithnoparameteranamenoparametera",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:10]: We already have a question on the pop method for this Python quiz.
Please remove this duplicate question.

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+  },
+  {
+    Question: "In Python, which of the following outputs 'camp'?",
+    Answer: "print('freecodecamp'[-4:])",
+    Distractor1: "print('freecodecamp'[8::-1])",
+    Distractor2: "print('freecodecamp'[8:11])",
+    Distractor3: "print('freecodecamp'[-4:10])",
+    Explanation:
+      "Here we use the negative index to start slicing 4th index from the end of the string.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:19]:
```suggestion
      "Here we use the negative index to start slicing at the 4th index from the end of the string.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Kunal-Garg-12** [comment]
Edited as required. Please verify

**Comment by Kunal-Garg-12** [comment]
Edited as required. Please verify

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**